src/pages/ArticleDetails.tsx

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable prettier/prettier */
+import React, { useState } from "react"
+import { RootStackScreen } from "navigation/NavigationTypes"
+import { Image, Linking } from "react-native"
+import { WebView } from "react-native-webview"
+import { ScrollPage } from "components/ScrollPage"
+import { usePalette } from "utils/colors"
+
+import { Asset } from "expo-asset"
+import { Roboto_400Regular } from "@expo-google-fonts/roboto"
+
+export const Article: RootStackScreen<"Article"> = props => {
+    const { isLight } = usePalette()
+    const article = props.route.params.article
+    const [webHeight, setWebHeight] = useState<number>(400)
+    const [refreshing, setRefreshing] = useState<boolean>(false)
+    let html: string[] = []
+    try {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+        html = JSON.parse(article.content)
+    } catch (error) {
+        console.log(error)
+    }
+
+    return (
+        <ScrollPage
+            navbarOptions={{ elevated: true }}
+            title={article.title}
+            subtitle={article.subtitle}
+            backdropElement={
+                <Image
+                    source={{
+                        uri: article.image,
+                    }}
+                    style={{
+                        width: "100%",
+                        height: "100%",
+                    }}
+                />
+            }
+            refreshControl={{
+                refreshing,
+                onRefresh: () => {
+                    setRefreshing(true)
+                    setTimeout(() => {
+                        setRefreshing(false)
+                    }, 2000)
+                },
+            }}
+        >
+            <WebView
+                onMessage={event => {
+                    console.log(event)
```

Comment:
non lasciare console log in giro a caso pls T.T

Reply:
Ops my bad :(